packages/mdc-tab-scroller/foundation.ts

```diff
@@ -238,27 +254,22 @@ export class MDCTabScrollerFoundation extends MDCFoundation<MDCTabScrollerAdapte
   }
 
   /**
-   * Internal increment scroll method
+   * Internal method to compute the increment scroll operation values.
    * @param scrollX The new scroll position increment
    */
-  private incrementScroll_(scrollX: number) {
+  private getIncrementScrollOperation_(scrollX: number) {
```

Comment:
Let's add a return type and include it in the Jsdoc comment.

Reply:
Done